src/components/uielements/CasePreview.astro

```diff
@@ -58,21 +58,23 @@ const { caseData, small } = Astro.props;
         <div class="flex flex-col">
           <span
             class="font-serif text-primary"
-            set:html={caseData?.metadata?.title}
+            set:html={caseData?.company?.name}
           />
           <h2
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
             set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData?.metadata?.image?.asset._id && (
+          caseData?.company?.logo?.dark?.asset._id && (
             <SanityImg
               loading="lazy"
-              src={caseData?.metadata?.image}
+              src={caseData?.company?.logo?.dark?.asset?._id}
               class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
               height={60}
-              alt={caseData?.metadata?.image?.asset?.altTexts?.[locale] || ""}
+              alt={
+                caseData?.company?.logo?.dark?.asset?.altTexts?.[locale] || ""
```

Comment:
did you check if the alt data is filled? I saw you did ... in the query. But I'm not sure if that query also get the altexts data

Reply:
yes altdata is filled:
![image](https://github.com/user-attachments/assets/46f9bb8b-c4d9-4374-bd29-2cf77b80a8a1)
![image](https://github.com/user-attachments/assets/8a24a690-1ef3-49e8-8bac-669bd924c239)